tests/spec/core/issues-notes-spec.js

```diff
@@ -160,6 +160,26 @@ describe("Core — Issues and Notes", function() {
     );
   });
 
+  it("should remove closed issues from spec", async () => {
```

Comment:
nit: `it(removes closed issues from spec...`. "should" is unnecessary.

Reply:
Everything now seems fine. But I have a doubt what if issue 1530 get reopened in the future again, in that case the test would fail

Reply:
here ya go: https://github.com/w3c/respec/issues/1548 😸